framework/auth/core.py

```diff
@@ -831,6 +905,10 @@ def get_confirmation_token(self, email, force=False):
                 # Old records will not have an expiration key. If it's missing,
                 # assume the token is expired
```

Comment:
@mfraezz 

I am not going to add this for CAS in this PR. The verification key is used immediately and distroyed. In addition, don't want CAS to block this.